README.md

```diff
@@ -53,18 +53,24 @@ See it up and running on http://localhost:8080
     
      1. Add to build.gradle
 >
- >apply plugin: 'com.bmuschko.tomcat'
+ >apply plugin: 'com.bmuschko.tomcat' 
```

Comment:
can you please refactor this part, to make it just like a code snippet
see an example here: https://gist.github.com/PurpleBooth/109311bb0361f32d87a2

Reply:
https://help.github.com/en/articles/creating-and-highlighting-code-blocks